alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/ChannelDistributionUIConfig.java

```diff
@@ -86,6 +86,11 @@ public ChannelDistributionUIConfig(ChannelKey channelKey, String label, String u
         return Stream.concat(configFields.stream(), channelDistributionFields.stream()).collect(Collectors.toList());
     }
 
+    @Override
+    public boolean hasFields() {
```

Comment:
Can we make this a little bit clearer? Some UIConfigs inherit fields, but don't defined any.

Reply:
Could we make it a final method that checks if `createFields()` is not empty?

Reply:
I wanted to avoid creating the fields just to throw them away and not use them.

Reply:
I feel our code is more bug prone if hard code true and false. If a user changes one method they have to remember the other one exists and maybe change it accordigly.